packages/esm-tb-app/src/config-schema.ts

```diff
@@ -18,6 +18,7 @@ export const configSchema = {
       tbTreatmentAndFollowUp: '1881304a-4854-4927-b0b1-a6231d61e43c',
       tptCaseEnrollment: 'dc6ce80c-83f8-4ace-a638-21df78542551',
       tptTreatmentAndFollowUp: '1ac3de3f-8fc2-43a7-addb-e805c393ecae',
+      tptTreatmentOutcome: 'dc6ce80c-83f8-4ace-a638-21df78542551',
```

Comment:
@lucyjemutai You don't need this. Treatment Outcome is part of the Case Enrolment encounter